manifests/deis-workflow-rc.yml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: deis-workflow
-          image: quay.io/deisci/workflow:v2-beta
+          image: quay.io/deis/workflow:v2-beta
```

Comment:
@sgoings no. we still want to track the stable builds.

@mboersma i disagree completely. It makes it extremely confusing that we are producing images in two different orgs. The images we have cut for alpha are in `deis` while images being produced right now are in `deisci`. It makes it difficult to go to the quay ui and see what has been updated and what hasn't. The upkeep will become a nightmare as we produce more versions of the beta and alpha releases. While I agree that jenkins testing images should be in the deisci org. Anything that can be used by us or a customer should be in the deis org.